docs: Add IJON Documents (#40)
fuzzuf/fuzzuf: docs/algorithms/ijon/implementation_en.md

# IJON implementation in fuzzuf

## Reference IJON Implementation

- Version: 2.51b-ijon
- Commit: https://github.com/RUB-SysSec/ijon/commit/56ebfe34709dd93f5da7871624ce6eadacc3ae4c

## Differences from the original implementation

Unless there are unintended discrepancies due to bugs, the original implementation has been completely reproduced and there are no differences between it and ours.
However, the functions of the original AFL, which are not implemented in fuzzuf AFL, cannot be used of course also in IJON.

## Important To-Do: Implementing Annotations

> It should be noted that annotations, which must be a major component of IJON, are not implemented in fuzzuf at present.

Reviewer: Something feels a bit off on the expression "must be" as well as the corresponding expression in Japanese docs. Aren't you sure that the annotation is a major component of IJON?
I'd say:
```suggestion
Be noted that annotation, which is one of the major components of IJON, is not implemented in fuzzuf at present.
```
